### ISSUE TITLE
Feature Request theme:clone

### ISSUE THREAD

**Issue by GaryCornell** [igaster/laravel-theme#58]
I think a good feature using the commands console would be to clone and rename a theme w/ assets so one could create a new theme from an already completed theme. I have seen and used this type of feature on several forum software apps such as Invision Power Board which I have created many mods for.

I am working on it and will post my code when I get there. Im thinking using the PHP copy() function and then further it with the PHP rename() function. Seems simple enough but am dissecting your current console commands to work it into the theme app specific.  I like the ease of theme::create / theme:package / theme:install and clone seems to be the next logical step.

Is it worth the trouble? Anyone think this will be useful?

**Comment by igaster** [comment]
Cloning a theme is as easy as making a copy of the views+assets folders and updating the theme.json file to point to the correct asset location. In what scenario would you prefere to clone a theme instead of extending it? Extending a theme will come with the benefit of staying always synced when for example the parent themes updates a js library or fixes a css error etc...

However It wouldn't be difficult to implement. I could put it on the roadmap :-)

**Comment by GaryCornell** [comment]
You're right it can done simply by making copies of those directories and renaming, lol, just as a developer who has pulled out many hairs over the years I strive for simplicity and running one command to do it is sweet to me. Also I am currently implementing this app into the database (caching templates, user theme select, and rebuilding views, etc), it reflects work I have done on IPB which can do everything in your command console from the admin panel with a click or two. Anyone whos used software like that knows the benefits of this method. Its not adding a great new feature but more of  how easy can I make this work...

No need to stress over it Ill work it up and post, if you deem it beneficial then be my guest and use it. 

One thing I am having difficulty with is my vendor views in my theme, its not showing my apps vendor folder views by the 3rd party vendor directory views and thats not working nicely in my app. I think it may be a path issue, sure Ill figure it out, I still have a few hairs left.

Thanks for the great app.

**Comment by igaster** [comment]
Yesterday I added some [documentatin about vendor paths](https://github.com/igaster/laravel-theme/wiki/9.-Namespaced-views), it might help. I have build an application for testing which covers many scenarios which I plan to publish soon,

Tip: If you want to debug themes, a quick way to display the Laravel's search paths (for views only) is:

    dd(app()->make('view.finder')->getPaths())

**Comment by GaryCornell** [comment]
Great! Ill have a look

**Comment by GaryCornell** [comment]
I added the debug line in my admin app layout and got the following

```
array:3 [▼
  0 => "/path_to/resources/views/smarty/smarty"
  1 => "/path_to/resources/views/smarty"
  2 => "/path_to/resources/views"
]
```
My theme is obviously named smarty. Is this proper behavior to show smarty/smarty in the array?

BTW the app I am using is arcanedev/Log-Viewer for admin side logs. The only relevant settings are in the log-viewer config
```
    /* -----------------------------------------------------------------------------
     |  Route settings
     | ------------------------------------------------------------------------------
     */
    'route'         => [
        'enabled'    => true,

        'attributes' => [
            'prefix'     => 'admin/log-viewer',

            'middleware' => ['web', 'admin'],
        ],
    ],
```

Everything else works fine
